modules/mobile/spec/request/gender_identity_request_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+require 'va_profile/models/gender_identity'
+require 'va_profile/demographics/service'
+
+RSpec.describe 'gender identity', type: :request do
+  include SchemaMatchers
+
+  let(:user) { FactoryBot.build(:iam_user) }
+  let(:time) { Time.zone.local(2022, 4, 8, 15, 9, 23) }
```

Comment:
Good catch, that was from me copying the original test

Reply:
I was just saying you didn't need to freeze to this specific time, but you're probably correct that you don't need timecop at all here because the request matches the VCR and you're not testing anything specific to time. Just don't forget to remove the timecop.return too.

Reply:
Ah got it. Removed the timecop.return